app/src/main/java/com/jerboa/model/SiteViewModel.kt

```diff
@@ -66,6 +66,8 @@ class SiteViewModel(private val accountRepository: AccountRepository) : ViewMode
 
                     if (!it.isAnon()) {
                         fetchUnreadCounts(GetUnreadCount(auth = it.jwt))
+                    } else {
+                        unreadCount = 0
```

Comment:
There is already a `isAnon` block above. And instead of manually trying to force an immutable var, trace down where unread count is read, and make it zero when its an anonymous account. Or creat a `getUnreadCount(isAnon: boolean)` type function.